scss/_buttons.scss

```diff
@@ -10,7 +10,7 @@
   @include rfs($btn-font-size, --#{$prefix}btn-font-size);
   --#{$prefix}btn-font-weight: #{$btn-font-weight};
   --#{$prefix}btn-line-height: #{$btn-line-height};
-  --#{$prefix}btn-color: #{$body-color};
+  --#{$prefix}btn-color: var(--#{$prefix}body-color);
```

Comment:
We should maybe introduce a Sass variable (`$btn-color`) here for better customization.
This variable could be set either to `null` or as just mentionned `var(--#{$prefix}body-color)`.